R/civis_ml.R

```diff
@@ -80,15 +84,51 @@
 #'  \code{sparse_ridge_regressor} \tab	\code{\link{civis_ml_sparse_ridge_regressor}} \tab regression \tab	\href{http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html}{Ridge} \tab \cr
 #'  \code{gradient_boosting_regressor}	\tab \code{\link{civis_ml_gradient_boosting_regressor}} \tab regression \tab \href{http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html}{GradientBoostingRegressor} \tab \code{n_estimators=500, max_depth=2} \cr
 #'  \code{random_forest_regressor}	\tab \code{\link{civis_ml_random_forest_regressor}} \tab regression \tab \href{http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html}{RandomForestRegressor} \tab \code{n_estimators=500} \cr
 #'  \code{extra_trees_regressor} \tab \code{\link{civis_ml_extra_trees_regressor}} \tab regression	\tab \href{http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html}{ExtraTreesRegressor} \tab \code{n_estimators=500} \cr
+#'  \code{multilayer_perceptron_regressor} \tab \tab regression \tab \href{https://github.com/civisanalytics/muffnn}{MLPRegressor} \tab \cr
+#'  \code{stacking_regressor} \tab \tab regression  \tab \href{https://github.com/civisanalytics/civisml-extensions}{StackedRegressor}\tab \cr
 #' }
 #' Model names can be easily accessed using the global variables \code{CIVIS_ML_REGRESSORS} and \code{CIVIS_ML_CLASSIFIERS}.
-#'
-#'
+#' @section Stacking:
+#'
+#' The \code{"stacking_classifier"} model stacks
+#' together the \code{"sparse_logistic"}, \code{"gradient_boosting_classifier"},
+#' and \code{"random_forest_classifier"} models, using altered defaults as
+#' listed for each in the \code{"Altered Defaults"} column of the table
+#' above. The models are combined using a pipeline
+#' containing a \href{http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html#sklearn.preprocessing.Normalizer}{Normalizer}
+#' step, followed by a \href{http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegressionCV.html}{LogisticRegressionCV}
+#' with \code{penalty='l2'} and \code{tol=1e-08}. The
+#' \code{"stacking_regressor"} works similarly, stacking together the
+#' \code{"sparse_linear_regressor"}, \code{"gradient_boosting_regressor"},
+#' and \code{"random_forest_regressor"} models, and combining them using
+#' \href{https://github.com/civisanalytics/civisml-extensions}{NonNegativeLinearRegression}.
+#'
+#' @section Hyperparameter Tuning:
+#' You can tune hyperparamters using one of two methods: grid search or
+#' hyperband. CivisML will perform grid search if you pass a list
+#' of hyperparameters to the \code{cross_validation_parameters} parameter, where list elements are
+#' hyperparameter names, and the values are vectors of hyperparameter
+#' values to grid search over. You can run grid search in parallel by
```

Comment:
This should say "hyperparameter optimization in parallel" rather than "grid search in parallel".

Reply:
(I just made this change in the civis-python docs, so they'll match)